src/routes/common/address-info/address-info.helper.ts

```diff
@@ -25,6 +25,10 @@ export class AddressInfoHelper {
    * The promise can be rejected if the address info cannot be retrieved for
    * any specified {@link source}
    *
+   * The function will try to get the address info from the provided sources
+   * in the order they are provided. If the address info cannot be retrieved
+   * from a source, the next source will be tried.
+   *
```

Comment:
This is not strictly related to the PR purpose, but I thought extra documentation would be beneficial as the function behavior is not obvious.